Templates/ManualTrigger-CSharp/metadata.json

```diff
@@ -0,0 +1,8 @@
+{
+    "defaultFunctionName": "ManualTriggerCSharp",
+    "description": "A C# function without triggers, inputs, or outputs",
+    "name": "ManualTrigger - C#",
+    "trigger": "MaunualTrigger",
+    "language": "C#",
+    "category": [ "Experimental" ]
```

Comment:
Had earlier moved them to experimental because of this issue #87. Moving both of them to core.